lnwallet/wallet.go

```diff
@@ -779,17 +780,44 @@ func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg
 	// funder in the attached request to provision the inputs/outputs
 	// that'll ultimately be used to construct the funding transaction.
 	if !ok {
+		var err error
```

Comment:
`lnwallet` changes should be separate from `lntest/itest` changes

Reply:
Moving it to previous commit.